core/src/code/instrumented.rs

```diff
@@ -30,6 +30,24 @@ use scale_info::{
     TypeInfo,
 };
 
+/// Instantiated section sizes for charging during module instantiation.
```

Comment:
Please comment here that it's not sizes in wasm binary, rather this is size of section representation in executor during module instantiation.

Reply:
By `Instantiated section size` I meant exactly  `size of section representation in executor during module instantiation`. 
If it's not clear I can rephrase it as you suggested.

Reply:
Updated comments